### ISSUE TITLE
Data from previous version not appearing

### ISSUE THREAD

**Issue by aliya-ryan** [digidem/mapfilter-desktop#12]
I was using an older version of Tizii Tizii which had all the data in it, but when I installed this version none of the data appears, which should be there.

**Comment by okdistribute** [comment]
Aye thanks for the tip. I haven't released the new version yet, and haven't tested this exact thing

**Comment by aliya-ryan** [comment]
Hey Karissa - I told Greg about this when I found it last week, he said he was going to investigate, but probably hasn’t done anything yet.

**Comment by aliya-ryan** [comment]
I have reinstalled version 0.5.0 as I really needed to see the data. It all appeared there as normal when I opened it up, so for some reason the new version is just not displaying it but it was still there somewhere.

**Comment by okdistribute** [comment]
Yeah, that's the best way forward. We're looking to move away from this desktop app over time

**Comment by aliya-ryan** [comment]
This is not a priority issue for the Wapichan - it just is a problem for me.
